github-adapter/src/main/java/be/xplore/githubmetrics/githubadapter/mappingclasses/GHAppInstallationAccessToken.java

```diff
@@ -1,6 +1,8 @@
 package be.xplore.githubmetrics.githubadapter.mappingclasses;
 
 public record GHAppInstallationAccessToken(
-        String token
+        String token,
+        String expires_at
 ) {
+    // 2016-07-11T22:14:10Z
```

Comment:
What does this comment mean?

Reply:
I forgot to add token invalidation to the AuthTokenInterceptor, has been added now, please check if its correct like this